fs-one-concepts.md

```diff
@@ -3,19 +3,36 @@
 Try your best to answer each question on your own before looking up the answer online. Once you're done writing your first answer, you can google the question and write the best answer you find.
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
+Enzyme is a JavaScript Testing utility for React that makes it easier to test your React Components' output. 
+You can also manipulate, traverse, and in some ways simulate runtime given the output.
 
 #### 2. What is the difference between dynamic and a static routes?
+Static Routing is also known as non-adaptive routing which doesn’t change routing table unless the network administrator changes or modify them manually.
+Dynamic routing is also known as adaptive routing which change routing table according to the change in topology.
 
 #### 3. What is a JSON API?
+JSON stands for JavaScript object notation and is used to interchange data.
```

Comment:
Good start here. Your Rails app side of Cat Tinder is a good example of a JSON API in that is an application that sends and receives JSON data.